include/seqan3/io/alignment_file/format_bam.hpp

```diff
@@ -694,7 +694,7 @@ inline void format_bam::write_alignment_record([[maybe_unused]] stream_type &  s
         if (ref_offset.has_value() && (ref_offset.value() + 1) < 0)
             throw format_error{detail::to_string("The ref_offset object must be >= -1 but is: ", ref_offset)};
 
-        seqan3::ostreambuf_iterator stream_it{stream};
+        std::cpp20::ostreambuf_iterator stream_it{stream};
```

Comment:
Why did you change this? We explicitly introduced our own ostreambuf_iterator because we had a significant performance improvement when exposing the buffer pointers and writing directly to the buffer.

Reply:
Edit: Sorry, I just saw that you are right `seqan3::ostreambuf_iterator` indeed is either ranges::ostreambuf_iterator or std::ostreambuf_iterator. It is the `seqan::fast_ostreambuf_iterator` I was talking about. I think it should be changed to  `seqan::fast_ostreambuf_iterator` then.

Reply:
good that I renamed it to avoid confusion :)

Reply:
This is a performance improvement and I would not want to do it in this PR. I think we should do measurements if we swap the implementations. I think this can be done by a student :)

Reply:
I think we should change it right away, why wait for an improvement. But rene is the reviwer so I'll leave this PR to him

Reply:
Yes and I/O is not part of this or the next quarter. So we can look at the performance when we work on that.

Reply:
I would really have the applications done with the other stuff that is on our plate.

Reply:
I'm not against it, because it is low-hanging fruit, but I think it is outside this PR.